src/__experimental__/components/radio-button/radio-button-group.component.js

```diff
@@ -38,12 +38,12 @@ const RadioButtonGroup = (props) => {
 
   return (
     <RadioFieldsetStyle legend={ legend }>
-      <StyledRadioButtonGroup
+      <div
         role='radiogroup'
```

Comment:
`role='radiogroup'` is not a valid HTML attribute. Let's get rid of the `div` all together and move `tagComponent` to `RadioFieldsetStyle`. Thanks @damianOfKyan!

Reply:
resolved in 59ca991